x-pack/solutions/observability/plugins/infra/public/pages/metrics/inventory_view/components/dropdown_button.tsx

```diff
@@ -55,13 +53,8 @@ export const DropdownButton = withEuiTheme((props: PropsWithTheme) => {
         border: theme?.euiTheme.border.thin,
       }}
     >
-      {showKubernetesInfo ? (
-        <KubernetesTour>
```

Comment:
It looks like this component isn't related to the guided onboarding that we're removing, so we shouldn't remove it. 
Looks like this is just used in `showKubernetesInfo` https://github.com/elastic/kibana/blob/6461b7e26a9afb2710353c0a84a67e4935504da4/x-pack/solutions/observability/plugins/infra/public/pages/metrics/inventory_view/components/waffle/waffle_inventory_switcher.tsx#L133 and doesn't depend on guided_onboarding

Reply:
so we should keep x-pack/solutions/observability/plugins/infra/public/pages/metrics/inventory_view/components/kubernetes_tour.tsx and this usage